### PR TITLE
ccb - create fixtures document to track subject changes

### PR DESCRIPTION
**Pull request by cameron-barrett**
In this PR I created a fixture file to be used in a later PR when a storybook is created for tracking subject updates.

### CODE REVIEW

File: frontend/src/fixtures/updatesFixtures.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const updatesFixtures = {
+  oneUpdate:
+  {
+    "subject_area" : "CMPSC",
+    "quarter_yyyyq": "20234",
```
Comment (pconrad) [inline frontend/src/fixtures/updatesFixtures.js:5]: Fixtures look good

File: frontend/src/index.css [hunk 2]
```diff
@@ -11,4 +11,8 @@ body {
     font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
       monospace;
   }
+
+  .color-nav{
+    background-color: rgb(0, 54, 96);
+  }
```
Comment (pconrad) [inline frontend/src/index.css:17]: This doesn't seem to belong in this PR.   Shouldn't this be a different PR?

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
           <AppNavbarLocalhost url={currentUrl} />
         )
       }
-      <Navbar expand="xl" variant="dark" bg="dark" sticky="top" data-testid="AppNavbar">
+      <Navbar expand="xl" variant="dark" className="color-nav" sticky="top" data-testid="AppNavbar">
```
Comment (pconrad) [inline frontend/src/main/components/Nav/AppNavbar.js:16]: This seems unrelated.  We try to keep a PR to a single topic.

I suggest you build two separate branches by using git cherry-pick to separate these into a pr for the fixtures, and a separate PR for the color change.  Each of those is worth 5 pts separately.

If you intended to put them into one PR, then your PR title and description need to spell that out; but many organizations frown on that (i.e. having two or more unrelated concerns in a single PR.)

**Review by pconrad — CHANGES_REQUESTED** [review]
Please consider splitting this into two PRs, or else modify the description/title.

if you intended only for the fixtures to be in this PR, I can show you how to do that with git magic.

File: frontend/src/main/components/Nav/AppNavbar.js (hunk 3 above)
Comment (cameron-barrett) [reply frontend/src/main/components/Nav/AppNavbar.js:16]: I have a separate PR for the navBar changes currently, I would just like to remove those two commits from this PR without breaking it. I tried to undo the two commits about the NavBar but it ended up deleting .js files needed to run the site. If there is an easy way to roll those out of this PR I would like to know how.

Comment (cameron-barrett) [reply frontend/src/main/components/Nav/AppNavbar.js:16]: Nevermind, I resolved this in a new PR.
